src/main/scala/inox/solvers/SolverFactory.scala

```diff
@@ -71,6 +71,7 @@ object SolverFactory {
     "smt-cvc4"     -> "CVC4 through SMT-LIB",
     "smt-z3"       -> "Z3 through SMT-LIB",
     "smt-z3-opt"   -> "Z3 optimizer through SMT-LIB",
+    "smt-z3:EXEC"  -> "Z3 through SMT-LIB with custom executable name",
```

Comment:
Nit: maybe use `<exec>` instead of `EXEC` for consistency with the CVC4 options: https://github.com/epfl-lara/inox/blob/93550f1bcbf459e4c5584a5a87dbd0fbcb124eab/src/main/scala/inox/solvers/smtlib/CVC4Solver.scala#L13

Reply:
Fixed, thanks